NitroxPatcher/Patches/Dynamic/Player_OnPlayerPositionCheat_Patch.cs

```diff
@@ -0,0 +1,20 @@
+using System.Reflection;
+using NitroxClient.GameLogic;
+using NitroxModel.DataStructures.Util;
+using NitroxModel.Helper;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+// This method sets the player's subroot to null, so send a packet accordingly
+public sealed partial class Player_OnPlayerPositionCheat_Patch : NitroxPatch, IDynamicPatch
+{
+    private static readonly MethodInfo TARGET_METHOD = Reflect.Method((Player t) => t.OnPlayerPositionCheat());
+
+    public static void Prefix(Player __instance)
+    {
+        if (__instance.GetCurrentSub() != null)
```

Comment:
`__instance.GetCurrentSub()` returns a MB and therefor should not be compared to null directly (it's a unity shenanigan).
You can use `if (__instance.GetCurrentSub())` instead of checking against null, kinda like in c++ with pointers.